code/modules/mob_spawn/corpses/mining_corpses.dm

```diff
@@ -222,6 +254,27 @@
 	if(prob(50))
 		neck = /obj/item/bedsheet/rd/royal_cape
 
+/datum/outfit/consumed_eskimo
+	name = "Legion-Consumed Eskimo"
```

Comment:
Could we just call them "travellers" or "explorers" something, it's less evocative but there is no cultural group on Earth who _wants_ to be called Eskimos so there probably also aren't any in space, and it would probably be weird to replace it with the name of a _different_ arctic earth cultural group too.

Reply:
there is lavaland eclousare-ruin with mobs called eskimos, based these off those.

Reply:
Oh yeah I don't much like that either, but opening a PR to remove words I don't like tends to be a very large headache 😅